Makefile

```diff
@@ -568,6 +570,13 @@ CC		:= $(CONFIG_CROSS_COMPILE)$(CONFIG_COMPILER)
 CPP		:= $(CC)
 CXX		:= $(CPP)
 GOC		:= $(CONFIG_CROSS_COMPILE)gccgo-7
+# We use rustc because the gcc frontend is experimental and missing features such
+# as borrowing checking
+ifneq ("$(origin CROSS_COMPILE)","undefined")
+RUSTC		:= rustc --target=$(CONFIG_CROSS_COMPILE)
```

Comment:
Shouldn't this be set to `LLVM_TARGET_ARCH`?

Reply:
Solved :dagger: